.github/workflows/ci.yml

```diff
@@ -34,6 +34,7 @@ jobs:
       run: |
         make cluster/prepare
         make test/e2e
+        make cluster/clean
```

Comment:
Why do we need this?

Reply:
IIRC (it's a while since I looked into this :)), there was some reason behind it. I believe the image E2E tests deploy the CRDs etc. implicitly, therefore we need to remove the previously created stuff.

Reply:
Ok, thanks!